llama_stack/cli/stack/build.py

```diff
@@ -271,7 +277,20 @@ def _run_stack_build_command_from_build_config(
         if return_code != 0:
             return
 
-        self._generate_run_config(build_config, build_dir)
+        if template_name:
+            # copy run.yaml from template to build_dir instead of generating it again
+            template_path = pkg_resources.resource_filename(
+                "llama_stack", f"templates/{template_name}/run.yaml"
+            )
+            os.makedirs(build_dir, exist_ok=True)
+            run_config_file = build_dir / f"{build_config.name}-run.yaml"
+            shutil.copy(template_path, run_config_file)
+            cprint(
+                f"You can now edit {run_config_file} and run `llama stack run {run_config_file}`",
```

Comment:
this is a bit of a dangerous message because it does not tell you what environment vars you need to also set? 

also, the long file name gets repeated -- can we only mention that file once?